NbMGui/src/main/java/nbmgui/Controller.java

```diff
@@ -76,6 +83,44 @@ void openFile(ActionEvent event) {
             outputView.appendText("Select a NoBeard object file\n");
     }
 
+    @FXML
+    void startProgram(ActionEvent event) {
+        setBreakpoints();
```

Comment:
This setBreakpoints() thing does not make me very happy. It is too complicated in my view point. Wouldn't it be more efficient to add some action handler to each Checkbox (via setOnAction), which is simply taking the address part of its string and its state (checked or unchecked) and then calls machine.addBreakpoint(...) or machine.removeBreakpoint()